core/src/stages/connect.cpp

```diff
@@ -59,8 +59,12 @@ Connect::Connect(const std::string& name, const GroupPlannerVector& planners) :
 	p.declare<MergeMode>("merge_mode", WAYPOINTS, "merge mode");
 	p.declare<moveit_msgs::msg::Constraints>("path_constraints", moveit_msgs::msg::Constraints(),
 	                                         "constraints to maintain during trajectory");
+	p.declare<float>("goal_tolerance", 0.02,
+	                 "L1 distance between goal state and end of trajectory state should be within this tolerance");
```

Comment:
nit: But is this actually the L2 distance (square root of L2 norm)? L1 would be the Manhattan distance. I think you can just remove the "L1" for what it's worth.

Reply:
It mentioned it was the L1 norm here - https://github.com/ros-planning/moveit2/blob/c9a9e407b0a27858c8eea838ccbbd389d7109f9e/moveit_core/robot_state/include/moveit/robot_state/robot_state.h#L1460
But it makes sense not to mention that here